jbmc/regression/jbmc/assume-inputs-interval/arg_assume_invalid2.desc

```diff
@@ -4,7 +4,7 @@ My.class
 ^EXIT=1$
 ^SIGNAL=0$
 ^Invalid User Input
-^Reason: interval must be of the form
+^Reason: argument must be .* of the form
```

Comment:
:pick: Can this be anything else than ` argument must be a sequence of intervals of the form`?